weblogic-azure-aks/src/main/arm/scripts/queryStorageAccount.sh

```diff
@@ -0,0 +1,49 @@
+export aksClusterRGName=$1
+export aksClusterName=$2
+export wlsDomainUID=$3
+
+export wlsDomainNS="${wlsDomainUID}-ns"
+export currentStorageAccount="null"
+
+# Connect to AKS cluster
+function connect_aks_cluster() {
+    az aks get-credentials \
+        --resource-group ${aksClusterRGName} \
+        --name ${aksClusterName} \
+        --overwrite-existing
+}
+
+function query_storage_account() {
+    echo "install kubectl"
+    az aks install-cli
+
+    echo "get pv, pvc"
+    pvcName=${wlsDomainUID}-pvc-azurefile
+    pvName=${wlsDomainUID}-pv-azurefile
+    
+    ret=$(kubectl -n ${wlsDomainNS} get pvc ${pvcName} | grep "Bound")
```

Comment:
This doesn't have to be fixed now, but I've noticed the pattern several times in this PR or performing a `kubectl get` and then doing a `grep` on the results rather then using the JSONPath support to get the exact field that you want (https://kubernetes.io/docs/reference/kubectl/jsonpath/).

Reply:
How long before some unfortunate customer includes "Bound" in the name of the PVC? :)

Reply:
For the first comment, we used `grep` in the older source code, and after them, we found JSONPath is more accurate．I will create another PR to enhance all of them.

Reply:
For the second comment, here, we would like to check if we need to create another Azure Storage Account for storage, we assume the PVC is bound. 
You are right, we should check pvc status and wait for it bound when it is created.